pom.xml

```diff
@@ -5,8 +5,155 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.epam.fazlyeva</groupId>
-    <artifactId>Kamilla_Fazlyeva_Homework</artifactId>
+    <artifactId>KamillaFazlyeva</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
 
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.source.version>1.8</java.source.version>
+        <java.target.version>1.8</java.target.version>
+
+        <testng.version>7.1.0</testng.version>
+        <testng.version>7.1.0</testng.version>
+        <selenium.version>3.141.59</selenium.version>
+        <webdrivermanager.version>3.8.1</webdrivermanager.version>
+        <assertj.version>3.15.0</assertj.version>
+        <common-io.version>2.6</common-io.version>
+
+        <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+        <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+    </properties>
+
+    <profiles>
+        <profile>
+            <id>Additive</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
Why do you decide make this profile active by default?

Reply:
It is a test requirement: "by default should work profile which runs add and subtract operations tests". Additive operations are summation and subtraction.